assignments/objects.js

```diff
@@ -49,16 +97,54 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+class Parent {
```

Comment:
Here a class was not necessary, in fact I like the use of the getter/setter approach here but I want you to take a look at this again and do it with only objects which you will utilize nesting to achieve. ... this is a great approach and I don't fault you but see if you can do it via using object nesting... I mean in theory JavaScript doesn't have classes and really they are just being emulated via the use of constructor functions and prototype inheritance.